doc/riverctl.1.scd

```diff
@@ -268,6 +268,69 @@ A complete list may be found in _/usr/include/linux/input-event-codes.h_
 	and is made available through the _XCURSOR_THEME_ and _XCURSOR_SIZE_
 	environment variables.
 
+## INPUT CONFIGURATION
+
+The _input_ command can be used to create a configuration rule for an input
+device identified by its _name_.
+
+Below find a lost of all device properties that can be configured. However note
+that not every input device supports every property.
+
+*input* _name_ *events* *enabled*|*disabled*|*disabled-on-external-mouse*
```

Comment:
What does `disabled-on-external-mouse` mean? Should we perhaps point users to libinput docs on these options?

Reply:
it means 'disable the device while an external mouse is plugged in', not sure if there is a man page instead of https://wayland.freedesktop.org/libinput/doc/latest/configuration.html